src/components/Navigator/NavigatorCard.vue

```diff
@@ -472,6 +476,22 @@ export default {
         this.$refs.scroller.scrollToItem(index);
       }
     },
+    // TODO: This is very much a test
```

Comment:
Is this comment still relevant?

Reply:
I dont think so. We will probably iron out any bugs we notice, but from my testing it was alright.